lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -1380,6 +1380,9 @@ void ResourceMap::loadDefaultConfiguration()
   addAsUnsignedInteger("AdaptiveDirectionalStratification-DefaultMaximumStratificationDimension", 3);
   addAsUnsignedInteger("AdaptiveDirectionalStratification-DefaultNumberOfSteps", 2);
 
+  // Analytical parameters //
+  addAsScalar("Analytical-defaultLimitStateTolerance", 0.01);
```

Comment:
must start with uppercase

Reply:
also it must be added to openturns.conf.in

Reply:
fixed